Classes/TYPO3/Party/Domain/Model/AbstractParty.php

```diff
@@ -24,52 +23,55 @@
  * @Flow\Entity
  * @ORM\InheritanceType("JOINED")
  */
-abstract class AbstractParty {
-
-	/**
-	 * A unidirectional OneToMany association (done with ManyToMany and a unique constraint) to accounts. This is
-	 * required to not have any dependencies from Account to AbstractParty (the other way round).
-	 *
-	 * @var Collection<\TYPO3\Flow\Security\Account>
-	 * @ORM\ManyToMany
-	 * @ORM\JoinTable(inverseJoinColumns={@ORM\JoinColumn(unique=true, onDelete="CASCADE")})
-	 */
-	protected $accounts;
-
-	/**
-	 * Constructor
-	 */
-	public function __construct() {
-		$this->accounts = new ArrayCollection();
-	}
+abstract class AbstractParty
+{
+    /**
+     * A unidirectional OneToMany association (done with ManyToMany and a unique constraint) to accounts. This is
+     * required to not have any dependencies from Account to AbstractParty (the other way round).
+     *
+     * @var Collection<\TYPO3\Flow\Security\Account>
```

Comment:
Does something speak against using the relative namespaces also in the collection / array syntax?